### ISSUE TITLE
confirm in View->on() only works with Callback, not with jsReload for example

### ISSUE THREAD

**Issue by PhilippGrashoff** [atk4/ui#503]
Hi there,

I tried the following (reload a view but only if user confirms):
```
$button_delete->on(
    'click', 
    (new \atk4\ui\jsReload($this->mainForm, ['action' => 'delete', 'delete_id' => $this->mainForm->model->get('id')])), 
    ['confirm' => $this->_tConfirmDelete->render()]);
```

This does not work - the reload is executed as soon as the button is clicked. View->on() only takes care of 'confirm' if a php callback is defined. It would be great if it also worked out of the box otherwise.

**Comment by romaninsh** [comment]
I agree, that would be a good addition.

**Comment by DarkSide666** [comment]
fixed by #642
